src/xo-app/vm/tab-network.js

```diff
@@ -318,9 +342,10 @@ export default class TabNetwork extends Component {
                         ? <Tooltip content={_('vifLockedNetworkNoIps')}>
                           <Icon icon='error' />
                         </Tooltip>
-                        : <Tooltip content={lockedNetwork && _('vifLockedNetwork')}>
-                          <Icon icon={lockedNetwork ? 'lock' : 'unlock'} />
-                        </Tooltip>}
+                        : lockedNetwork ? <Tooltip content={_('vifLockedNetwork')}>
```

Comment:
And you're testing twice for `lockedNetwork`, it should be possible to simplify that?